### PR TITLE
Add quick temporary patch for Authorization error

### PR DESCRIPTION
**Pull request by raisedadead**
Refer's #8923

### CODE REVIEW

**Comment by raisedadead** [comment]
@BerkeleyTrue this will only block new email signups and they will also get a link in their email after this is merged.

I am trying to come up with a more elegant solution.

@QuincyLarson : I agree that we should be showing minimal notifications (possibly only once) for verification.

**Comment by raisedadead** [comment]
Current un-verified users are not going to be affected.

File: server/boot/a-extendUser.js [hunk 1]
```diff
@@ -87,14 +87,10 @@ module.exports = function(app) {
...
       if (err) { return next(err); }
-      return req.logIn(user, function(err) {
-        if (err) { return next(err); }
-
-        req.flash('success', {
-          msg: [ "Welcome to Free Code Camp! We've created your account." ]
-        });
-        return res.redirect(redirect);
+      req.flash('success', {
+        msg: [ "Welcome to Free Code Camp! We've created your account." ]
```
Comment (BerkeleyTrue) [inline server/boot/a-extendUser.js]: I don't think we should say 'welcome' if they are not logged in

**Comment by BerkeleyTrue** [comment]
Please update title

**Pull request merged**